src/targets/gpu/lowering.cpp

```diff
@@ -236,11 +240,37 @@ struct miopen_apply
             {
                 check_shape(s, insert_precompile_op(it));
             }
+            else if(attrs.contains("custom_op") and attrs["custom_op"].get_bool())
```

Comment:
This should be `if(attrs.contains("target"))`.

Reply:
Done